viewport/gui_in_3d/gui_3d.gd

```diff
@@ -1,102 +1,156 @@
 extends Spatial
-
 # Member variables
 # The size of the quad mesh itself.
-# NOTE: Do not apply the scale of the MeshInstance node, just the scale of the quad mesh!
-export (Vector2) var quad_mesh_size = Vector2(3, 2)
-# The scale of the quad node. It is assumed that the node is scaled evenly across the X, Y, and Z axes.
-export (float) var quad_node_scale = 1
-# The position of the last processed input touch/mouse event.
-var prev_pos = null
-# The last non-empty click_pos position. We need this to simulate drag events.
-var last_click_pos = null
-# The viewport we want to interact with.
-var viewport = null
-# A empty Vector3 used for comparison.
-var empty_vector = Vector3(0,0,0)
+var quad_mesh_size
+# Indentify if the mouse is inside the Area
+var mouse_inside = false
+# Identify if the mouse was pressed inside the Area
+var mouse_held = false
+# The last non-empty mouse position. Used when dragging outside of the box.
+var last_mouse_pos3D = null
+# The last processed input touch/mouse event. To calculate relative movement.
+var last_mouse_pos2D = null
+# Most used nodes
+onready var node_viewport = $Viewport
+onready var node_quad = $Quad
+onready var node_area = $Quad/Area
+onready var node_collision = $Quad/Area/CollisionShape
+
+func _ready():
+	node_area.connect("mouse_entered", self, "_mouse_entered_area")
+	print(node_quad.get_surface_material(0).params_billboard_mode)
```

Comment:
Yet another debug I forgot laying around X__X

Disabling `_proccess` is actually a very good idea. I was struggling to avoid as much processing as possible, but forgot I could just disable the entire function.